### PR TITLE
Release 0.13.0

### PR DESCRIPTION
**Pull request by davkal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,161 @@
...
+  [#964](https://github.com/weaveworks/scope/pull/964)
+- Details: only show important metadata by default, expand the rest
+	[#946](https://github.com/weaveworks/scope/pull/946)
+- Reorder the children tables in the details panel by importance
+	[#941](https://github.com/weaveworks/scope/pull/941)
+- Shorten docker container and image IDs in the details panel.
+	[#930](https://github.com/weaveworks/scope/pull/930)
+- shorten some details panel labels which were truncated
+	[#940](https://github.com/weaveworks/scope/pull/940)
+- Feed sparkline data via buffer to keep them moving
```
Comment (tomwilkie) [inline CHANGELOG.md]: This could do with more explaining I think.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,161 @@
...
+	[#919](https://github.com/weaveworks/scope/pull/919)
+- Periodically check for newer versions of scope.
+	[#907](https://github.com/weaveworks/scope/pull/907)
+- Rename Applications -> Process, sort topologies by rank.
+	[#866](https://github.com/weaveworks/scope/pull/866)
+- Rename 'by hostname' to 'by dns name'
+	[#856](https://github.com/weaveworks/scope/pull/856)
+- Add container uptime and restart count to details panel.
+	[#853](https://github.com/weaveworks/scope/pull/853)
+- Use connection directions from conntrack (no UI support yet)
```
Comment (tomwilkie) [inline CHANGELOG.md]: Ui does support this (it lays things out based on the direction of the edges).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,161 @@
...
+	[#967](https://github.com/weaveworks/scope/pull/967)
+
+Performance enhancements:
```
Comment (tomwilkie) [inline CHANGELOG.md]: Move to top?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,161 @@
...
+	[#918](https://github.com/weaveworks/scope/pull/918)
+- Minimize race between proc's net/tcp{,6} and fd/*
+	[#905](https://github.com/weaveworks/scope/pull/905)
+- Fix reading of network namespace inodes
+	[#898](https://github.com/weaveworks/scope/pull/898)
+
+Bug fixes:
+- Use TCP for weave dns to fix autoclustering
+  [#1038](https://github.com/weaveworks/scope/pull/1038)
+- Add ping/pong to websocket protocol
```
Comment (tomwilkie) [inline CHANGELOG.md]: ... to prevent websocket connections being dropped when traversing loadbalancers.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,161 @@
...
+	[#975](https://github.com/weaveworks/scope/pull/975)
+- don't render any metrics/metadata for uncontained node
+	[#956](https://github.com/weaveworks/scope/pull/956)
+- update go-dockerclient to fix bug with docker 1.10
+	[#952](https://github.com/weaveworks/scope/pull/952)
+- Show nice column labels when no children have metrics
+	[#950](https://github.com/weaveworks/scope/pull/950)
+- Fixes process-by-name layout with ./foo and /foo nodes
+	[#948](https://github.com/weaveworks/scope/pull/948)
+- Deal with starting / stopping weave
```
Comment (tomwilkie) [inline CHANGELOG.md]: ... whilst scope is running.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,161 @@
...
+	[#867](https://github.com/weaveworks/scope/pull/867)
+- Remove host links that link to themselves in details panel
+	[#917](https://github.com/weaveworks/scope/pull/917)
+- Just show the untruncated label in the tooltip on children
+	[#911](https://github.com/weaveworks/scope/pull/911)
+- Increase timeout, remove double close to fix TestAppClientDetails
```
Comment (tomwilkie) [inline CHANGELOG.md]: We usually omit test improvements from the release notes.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,161 @@
...
+	[#882](https://github.com/weaveworks/scope/pull/882)
+- Fixes host node/details panel color mismatch
+	[#880](https://github.com/weaveworks/scope/pull/880)
+
+Build improvements and cleanup:
+- Disable checkpointing in tests.
+	[#1031](https://github.com/weaveworks/scope/pull/1031)
+- Document how to obtain profiles without `go tool pprof`
+	[#993](https://github.com/weaveworks/scope/pull/993)
+- Don't log expected websocket errors
```
Comment (tomwilkie) [inline CHANGELOG.md]: not a build improvement, more a bug fix.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,161 @@
...
+	[#1024](https://github.com/weaveworks/scope/pull/1024)
+- Turn off GC for builds.
+	[#1023](https://github.com/weaveworks/scope/pull/1023)
+- Use short URL for scope download
```
Comment (tomwilkie) [inline CHANGELOG.md]: Not a build improvement - a docs fix.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,161 @@
...
+	[#892](https://github.com/weaveworks/scope/pull/892)
+- Stop details cards popping up over the terminal.
+	[#882](https://github.com/weaveworks/scope/pull/882)
+- Fixes host node/details panel color mismatch
+	[#880](https://github.com/weaveworks/scope/pull/880)
+
+Build improvements and cleanup:
+- Disable checkpointing in tests.
+	[#1031](https://github.com/weaveworks/scope/pull/1031)
+- Document how to obtain profiles without `go tool pprof`
```
Comment (tomwilkie) [inline CHANGELOG.md]: Not a build thing, a docs fix.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,161 @@
...
+	[#993](https://github.com/weaveworks/scope/pull/993)
+- Don't log expected websocket errors
+	[#1024](https://github.com/weaveworks/scope/pull/1024)
+- Turn off GC for builds.
+	[#1023](https://github.com/weaveworks/scope/pull/1023)
+- Use short URL for scope download
+	[#1018](https://github.com/weaveworks/scope/pull/1018)
+- Bump template name to get latest version of docker.
+	[#998](https://github.com/weaveworks/scope/pull/998)
+- added note about docker and go dependency to the readme
```
Comment (tomwilkie) [inline CHANGELOG.md]: docs

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,161 @@
...
+	[#966](https://github.com/weaveworks/scope/pull/966)
+- Overwrite /etc/weave/apps, because it might already exist
```
Comment (tomwilkie) [inline CHANGELOG.md]: not a build improvement, but a bug fix.

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,161 @@
...
+	[#959](https://github.com/weaveworks/scope/pull/959)
+- Fixes building scope outside of a container.
+	[#901](https://github.com/weaveworks/scope/pull/901)
+- log a warning when reporters or taggers take too long to generate
```
Comment (tomwilkie) [inline CHANGELOG.md]: not a build thing, a bug fix.

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,161 @@
...
+	[#944](https://github.com/weaveworks/scope/pull/944)
+- minor refactor of backend metadata and metric rendering
```
Comment (tomwilkie) [inline CHANGELOG.md]: ditto.

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,161 @@
...
+	[#920](https://github.com/weaveworks/scope/pull/920)
+- Add some tests, and a zero-value for report.Sets
```
Comment (tomwilkie) [inline CHANGELOG.md]: ditto

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,161 @@
...
+	[#903](https://github.com/weaveworks/scope/pull/903)
+- Don't need sudo when DOCKER_HOST is tcp.
+	[#888](https://github.com/weaveworks/scope/pull/888)
+- Disable npm progress to speed up build
+	[#894](https://github.com/weaveworks/scope/pull/894)
+- Update readme and images.
```
Comment (tomwilkie) [inline CHANGELOG.md]: docs fix.

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,3 +1,161 @@
...
+	[#885](https://github.com/weaveworks/scope/pull/885)
+- refactoring deepequal to satisfy linter
+	[#890](https://github.com/weaveworks/scope/pull/890)
+- Update approach to trigger signal dumps
```
Comment (tomwilkie) [inline CHANGELOG.md]: bug fix.

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,3 +1,160 @@
...
+	[#894](https://github.com/weaveworks/scope/pull/894)
+- refactoring deepequal to satisfy linter
+	[#890](https://github.com/weaveworks/scope/pull/890)
+
+Documentation:
+- Document how to obtain profiles without `go tool pprof`
+	[#993](https://github.com/weaveworks/scope/pull/993)
+- Use short URL for scope download
+	[#1018](https://github.com/weaveworks/scope/pull/1018)
+- added note about docker and go dependency to the readme
```
Comment (2opremio) [inline CHANGELOG.md]: Uppercase? (sorry OCD :) )

**Comment by 2opremio** [comment]
LGTM, I have tested in our k8s cluster and the ECS demo. Looking beautiful.

I have run again into https://github.com/weaveworks/scope/issues/868 and https://github.com/weaveworks/scope/issues/827 though. Marked them as 0.14

**Comment by tomwilkie** [comment]
/summon @pidster for lgtm'ing the release.

I still have testing to do, but it's going well so far.

**Comment by tomwilkie** [comment]
Probably want to include this in the release https://github.com/weaveworks/scope/pull/1052 (PR is targeting this release branch)

**Comment by tomwilkie** [comment]
I'm LGTM on this release now

**Comment by paulbellamy** [comment]
LGTM now

**Comment by pidster** [comment]
LGTM
